docs/api/config-manager.md

```diff
@@ -263,6 +263,14 @@ Looks up the definition of a given device in the configuration DB. It is not nec
 
 For details on the available properties, refer to the [config file documentation](development/config-files.md).
 
+### `lookupDevicePreserveConditions`
+
+```ts
+lookupDevicePreserveConditions(manufacturerId: number, productType: number, productId: number, firmwareVersion?: string): Promise<ConditionalDeviceConfig | undefined>;
+```
+
+Like `lookupDevice`, but does not evaluate `$if` conditions. The resulting object will contain all settings that are defined regardless if they apply or not.
```

Comment:
Maybe add here, the slight difference, with how the params are delivered.
Each param object being wrapped in an array block (1 object in the array for each condition assumably?)
where as `lookupDevice `returns an unwrapped object for each param

My best effot...

------------------------------
Like `lookupDevice`, but does not evaluate `$if` conditions. The resulting object will contain all settings that are defined regardless if they apply or not. Each Association Group, Config Parameter and Parameter Option, will include a `condition` statement, providing one has been provided.

Each Config parameter will come wrapped in an array (1 for each condition)

Reply:
I've opted to include the types instead.